docs/source/changelog.rst

```diff
@@ -18,6 +18,7 @@ Changelog
         * Update variable type inference to better check for string values (:pr:`683`)
     * Changes
         * Moved dask, distributed imports (:pr:`634`)
+	* Added to Featuretools plugins (:pr:`699`)
```

Comment:
might as well mention that it's auto normalize specifically. also, this should be under "Enhancements" since it's new functionality!

Reply:
also add yourself to contributors